src/main/java/de/zalando/zmon/dataservice/data/MetricTiers.java

```diff
@@ -0,0 +1,98 @@
+package de.zalando.zmon.dataservice.data;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import de.zalando.zmon.dataservice.TokenWrapper;
+import de.zalando.zmon.dataservice.components.DefaultObjectMapper;
+import de.zalando.zmon.dataservice.config.DataServiceConfigProperties;
+import org.apache.http.client.fluent.Executor;
+import org.apache.http.client.fluent.Request;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.*;
+
+@Component
+public class MetricTiers {
+    public static final Logger LOG = LoggerFactory.getLogger(MetricTiers.class);
+
+    private final ObjectMapper objectMapper;
+    private final Executor executor;
+    private final TokenWrapper wrapper;
+    private final String hostname;
+
+    private Integer ingestMaxCheckTier;
```

Comment:
explicitly initialize to 0 so it's clear that by default all tiers are allowed

Reply:
see constructor